styleguide/screens/Design/Responsive.js

```diff
@@ -0,0 +1,144 @@
+        </T>
+      </ol>
+      <T elm="p" className={ m.mtr }>
+        When working “bottom up”, it should apparent become that you need to add a media query when
+        the page or feature begins to look and feel rubbish. Look to increase the window size until
+        this point, add a media query which adds additional styles which improve the page or
+        feature’s appearance, then rinse and repeat. Use common sense with doing this, as we don’t
+        want to add too many media queries at the expense of bloat in our code.
+      </T>
+      <Specimen
+        classNames={ {
+          root: m.mtr,
+          specimenContainer: m.par,
+        } }
+        code={ dedent`
+          .container {
+            display: inline-block;
+            width: 100%;
+          }
+
+          @media(min-width: 30rem) {
+            .container {
+              width: 50%;
+            }
+          }
+        ` }
+      >
+        <H level={ 3 }>Example</H>
+      </Specimen>
+    </D>
+    <D>
+      <H level={ 2 }>Grid system</H>
+      <T elm="p" className={ m.mtr }>
+        Unlike many patterns on the web, we don’t follow a specific grid. We take the approach that
+        every layout is unique, and deserves a unique set of properties. This allows us to be much
+        more flexible, and design for content at hand.
+      </T>
+      <T elm="p" className={ m.mtr }>
+        For common patterns, such as splitting a container in half,
+        we used <A href="http://lostgrid.org/">Lost grid</A> to do the work for us. This shouldn’t
+        be used for positioning on the page, and only for breaking content into common column sizes.
+      </T>
+      <Specimen
+        classNames={ {
+          root: m.mtr,
+          specimenContainer: m.par,
+        } }
+        code={ dedent`
+          .outer {
+            lost-utility: clearfix;
+          }
+
+          .inner {
+            lost-column: 1/2;
+          }
+      ` }
+      >
+        <H level={ 3 }>Example</H>
+      </Specimen>
+    </D>
+    <D>
+      <H level={ 2 }>Responsive images</H>
+      <T elm="p" className={ m.mtr }>
+        As our designs are flexible, and transform based on screen size, we need our images to
+        fluidly scale. We can do this by filling a space and cropping a selection of the image,
+        or by constraining the height/width of an image to scale by retaining it’s aspect ratio.
+        Documentation <Link to="/patterns/fitted-image" className={ scaffoldCss.link }>here</Link>.
+      </T>
+    </D>
+    <D>
+      <H level={ 2 }>Screen size over devices</H>
+      <T elm="p" className={ m.mtr }>
+        Rather than building for ‘devices’ we approach screen size as a whole. So if content doesn’t
+        look good as your screen gets bigger, add a media query to fix it.
+      </T>
+      <T elm="p" className={ m.mtr }>
+        Note: It’s important to note that ‘common’ screen sizes don’t exist. And we can never assume
```

Comment:
I'm assuming this is from the paper doc, but I think these should be `"` not `‘` and `’`